src/include/platform/internal/GenericPlatformManagerImpl.ipp

```diff
@@ -193,6 +193,7 @@ void GenericPlatformManagerImpl<ImplClass>::_RemoveEventHandler(PlatformManager:
 template <class ImplClass>
 void GenericPlatformManagerImpl<ImplClass>::_HandleServerStarted()
 {
+    mServerStarted                                    = true;
```

Comment:
Note that this is not actually strictly related to whether the _event loop_ is running.

Reply:
Undid these changes.